### DISCUSSION TITLE
Am I using the Tag Downloader wrong?

### DISCUSSION THREAD

**Discussion by AkitoCL** [TumblThreeApp/TumblThree#365] [Q&A (Question & Answer)]
I'm using the latest version of TumblrThree to download specific tagged posts from https://animarchive.tumblr.com but I'm just not getting enough downloads.
For instance the #anime-v tag should donwload at least 392 posts according to the archive but I just got 26 downloads.

Am I setting things wrong?
![image](https://user-images.githubusercontent.com/24996671/101999923-7b99da80-3cc0-11eb-8565-dc99601425ee.png)

**Comment by thomas694** [comment]
Yes, you have to use the text form like it's written on the blog page under a post (or in their internal json structure), in your case "anime v" (without hyphen and quotation marks).

Don't use the page title/header after e.g. a tag search as this can be showing the wrong spelling. See the page header in [https://www.tumblr.com/tagged/anime-v](https://www.tumblr.com/tagged/anime-v) and [https://www.tumblr.com/tagged/anime+v](https://www.tumblr.com/tagged/anime+v) (this is the actual link of the tag). The correct tag spelling is shown under the posts, although both urls show the same tagged posts but little different sidebar content.

**Comment by thomas694** [comment]
The issue has been closed. You can still comment. Feel free to ask for reopening the issue if needed.
